frontend/gpay_agent/lib/agent_views/fetch_store.dart

```diff
@@ -1,116 +1,178 @@
+                              color: Colors.blue,
+                              child: new Text("Submit"),
+                            ),
+                          ),
+                        ),
+                      ]
                   ),
                 ),
-                Text("OR"),
-                /// Calls [QrScan] user interface to fetch store using QR code
-                ButtonTheme(
-                  minWidth: 200,
-                  height: 50,
-                  child: RaisedButton(
-                  onPressed: () {
-                    Navigator.push(
-                      context,
-                      MaterialPageRoute(
-                          builder: (context) => QrScan(),
+                SizedBox(height: 20,),
+                Text("Method 2"),
+                /// Calls [_scan] function to scan QR code of store
+                Container(
+                  padding: EdgeInsets.all(15),
+                  decoration: BoxDecoration(
+                    border: Border.all(color: Colors.grey),
+                  ),
+                  child: Column(
+                    crossAxisAlignment: CrossAxisAlignment.start,
+                    children: <Widget>[
+                      Text(
+                        "Scan QR code",
+                        textAlign: TextAlign.left,
+                        style: TextStyle(
+                          fontWeight: FontWeight.bold,
+                        ),
+                      ),
+                      SizedBox(height: 8,),
+                      Text(
+                        "QR Code is generated when the merchant registers the store using merchant app",
+                      ),
+                      SizedBox(
+                        child: Image.memory(bytes),
                       ),
-                    );
-                  },
-                    textColor: Colors.white,
-                    color: Colors.blue,
-                    child: new Text("Scan QR Code"),
+                      SizedBox(height: 8,),
+                      Center(
+                        child: ButtonTheme(
+                          minWidth: 200,
+                          height: 50,
+                          child: RaisedButton(
+                              onPressed: _scan,
+                              textColor: Colors.white,
+                              color: Colors.blue,
+                              child: new Text("Scan QR Code"),
+                          ),
+                        ),
+                      ),
+                    ]
                   ),
                 ),
+                SizedBox(height: 30,),
                 Image.asset(
                   "assets/agent_beginning_images/GPay_logo_rectangle.png",
-                  width: 200,
-                  height: 100,
+                  height: 40,
                 )
               ],
             )
+        );
+  }
+  /// Fetches text of QR code by calling scan() API of qr_scan plugin.
```

Comment:
new line before new method

Reply:
Added and the merging part we are shifting to new branch because there are many small things to modify.